### ISSUE TITLE
Pre request variables ignored within post url

### ISSUE THREAD

**Issue by lrotermund** [usebruno/bruno#2664]
### I have checked the following:

- [X] I use the newest version of bruno.
- [X] I've searched existing issues and found nothing related to my issue.

### Describe the bug

Context: For good integration of Bruno in our large team, we maintain URL variables like `/foo/{{foo_id}}/bar` as 'Pre Request' variables, which we commit empty and do not overwrite.

Since some update that I can no longer specify, this functionality is broken and variables stored in the pre-request are no longer set and are simply submitted empty instead. Staying with the previous example

- Post url: `/foo/{{foo_id}}/bar`
- Pre-request **foo_id**: `123`
- Actual requested URL: `/foo//bar`

Was this behaviour intentionally patched?

### .bru file to reproduce the bug

```
meta {
  name: Buggy pre variables test
  type: http
  seq: 1
}

post {
  url: https://google.com/{{api}}
  body: json
}

headers {
  Content-Type: application/json
}

body:json {
  {
    "q": "{{query}}"
  }
}

vars:pre-request {
  api:
  query:
}
```

### Screenshots/Live demo link

1. shows the use of the defined 'Pre Request' variable 'api'.
2. shows that the pre-request variable 'api' has been set.
3. shows the timeline of the request and the empty replaced variables.
4. same as 3.
5. shows the current bruno version `v1.21.0`

![image](https://github.com/user-attachments/assets/698ac498-1a1e-4816-8e72-54c83aca57c3)

**Comment by Its-treason** [comment]
This is a duplicate of: #2627 it should be fixed in the next version when #2651 is merged

**Comment by lrotermund** [comment]
@Its-treason Thanks for the feedback, sorry I missed it when searching the open issues. I'm looking forward to the fix, as bruno is barely usable for us at the moment.
